app/controllers/ApplicationController.scala

```diff
@@ -200,6 +216,20 @@ trait ApplicationController extends BaseController {
       }
   }
 
+  def retrieveAnalysisExerciseInfo(applicationId: String): Action[AnyContent] = Action.async {
```

Comment:
I was confused when reading this because of "info" and "meta data", they both sounded like the same thing - could you rename this endpoint to perhaps retrieveAnalysisExerciseId?

Reply:
`retrieveAnalysisExerciseId` implies that all I'm retrieving is the file id but it could have more attributes right?

Reply:
I see that assessmentCentreTests returns an object. The two endpoints are still confusingly named, what's a better solution?

Call the second one analysisExerciseFileMetadata perhaps? Also pedantry shoutout, metadata is one work so no capital D in camel case :-)

Reply:
Renamed the other function